src/client/mvt/feature-decoder.js

```diff
@@ -8,6 +8,18 @@ export class Polygon {
     }
 }
 
+export function decodeLines(geom, mvt_extent) {
```

Comment:
```js
export function decodeLines(geom, mvt_extent) {
    return geom.map(l => l.map(point => (2 * point.x / mvt_extent - 1, 2 * (1 - point.y / mvt_extent) - 1)));
}
```

?

Reply:
Not exactly, that returns a different kind of structure, but I've simplified the function a little bit